src/plib/library/DomainConnect.php

```diff
@@ -26,7 +32,7 @@ private function getData()
             try {
                 $client = new \Zend_Http_Client($url);
                 $response = $client->request(\Zend_Http_Client::GET);
-            } catch (\Zend_Uri_Exception $e) {
```

Comment:
I think `Zend_Uri_Exception` was added for the case when _domainconnect TXT contains bad URL, e.g. it matched a wildcard record. Could you test your code with the domain `bad.ekazakov.ru`